config/dev.exs

```diff
@@ -31,6 +31,9 @@ config :mindwendel, MindwendelWeb.Endpoint,
     ]
   ]
 
+config :mindwendel, :options,
+  brainstorming_teasers: System.get_env("BRAINSTORMING_TEASERS") || "true"
```

Comment:
I recommend to map this env var to a boolean value that says whether the feature is enabled or not.

`config :mindwendel, :options, feature_brainstorming_teasers: !Enum.member?(["", "true"], String.trim(System.get_env("BRAINSTORMING_TEASERS") || "")`

This also considers nil, empty string an whitespace edge cases.